benchmarks/cpp/nvfuser/CMakeLists.txt

```diff
@@ -20,6 +20,7 @@ if(USE_CUDA)
     softmax_backward.cpp
```

Comment:
Note to myself: I have split this file out and merged separately in https://github.com/csarofeen/pytorch/pull/2007